watertap/property_models/NaCl_prop_pack.py

```diff
@@ -631,24 +631,26 @@ def rule_mole_frac_phase_comp(b, j):
             self.params.component_list, rule=rule_mole_frac_phase_comp
         )
 
-    def _molality_comp(self):
-        self.molality_comp = Var(
+    def _molality_phase_comp(self):
+        self.molality_phase_comp = Var(
             ["NaCl"],
```

Comment:
Here, and in lots of other places, we also need to update the indexing set (e.g., in this example it should be `[["Liq"],["NaCl"]]`.

Otherwise this will be the source of even further confusion.

Reply:
I agree. If we change the name to include the `phase` qualifier, then we need to include said index for the variable.

Reply:
Done